src/main/java/org/folio/inventory/storage/external/ExternalStorageModuleItemCollection.java

```diff
@@ -130,8 +130,8 @@ protected JsonObject mapToRequest(Item item) {
       ? item.id
       : UUID.randomUUID().toString());
 
-    if(item.getStatus().getString(Status.NAME_KEY) != null) {
-      itemToSend.put("status", item.getStatus());
+    if(item.getStatus().getName() != null) {
+      itemToSend.put("status", item.getStatus().getJson());
```

Comment:
Rather than using `getJson` here, maybe we could do the serialization in this class instead?

Reply:
Serialization is performed in two places: here and in `ItemRepresentation`. Should I create a representation class instead and move serialization to this class?

Reply:
I think I made a mistake trying to combine those too.

I think we want to head in the direction of serialization to storage and serialization back to the client as two separate activities.

I think the serlialization to storage can be included here, or use it's own class (I likely prefer the latter)

Reply:
I've extracting serialization and deserialization to separate class.